docs/content/users/usage/commands.md

```diff
@@ -1447,7 +1447,10 @@ ddev xhprof off
 Run [`yarn` commands](https://yarnpkg.com/cli) inside the web container in the root of the project (global shell host container command).
 
 !!!tip
-    Use `--cwd` for another directory.
+    Use `--cwd` for another directory, or you can change directories to the desired directory and `ddev yarn` will act on the same relative directory inside the container.
+
+!!!tip
+    If you want latest yarn versions, set `corepack_enable: true` in `.ddev/config.yaml` or `ddev config --corepack-enable`
```

Comment:
```suggestion
    If you want latest `yarn` versions, set `corepack_enable: true` in `.ddev/config.yaml` or `ddev config --corepack-enable`
```

Reply:
Instead wrapping the word Yarn in single quotes and making it a command i would vote to make it upper case and follow the wordmark which is used https://yarnpkg.com so it becomes a name/label

```suggestion
    If you want latest Yarn versions, set `corepack_enable: true` in `.ddev/config.yaml` or `ddev config --corepack-enable`
```
and since it sounds slightly odd wanting several of the latest versions why not slightly adjust the first part of the sentence to the gist of the third paragraph over here: https://yarnpkg.com/corepack

```suggestion
    If you want to define your Yarn version on a per project basis, set `corepack_enable: true` in `.ddev/config.yaml` or `ddev config --corepack-enable`
```

Reply:
Thanks, good stuff. Since "yarn" is a regular word, I don't think it needs to be added to spellcheck list.